Assets/Scripts/Util/AudioUtil.cs

```diff
@@ -5,9 +5,13 @@
 
 using UnityEngine;
 using System.Collections;
+using System.Text.RegularExpressions;
+using k = Global;
 
 public static class AudioUtil 
 {
+    static Regex idRgx = new Regex(".+" + k.JOIN_CHAR + ".+");
```

Comment:
I don't know enough about Regex to properly comment, but consider moving these substrings into Global